deploy/Install-MRP-app.sh

```diff
@@ -1,30 +1,31 @@
-﻿#!/bin/bash
+#!/bin/bash
+
+# Auto-accept license for Java
+echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections
 
 # Install packages
+add-apt-repository ppa:webupd8team/java
```

Comment:
Can you add -y in the end so it won't ask for confirmation:
add-apt-repository ppa:webupd8team/java -y